### PR TITLE
Change raw capacity tool tips message

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
https://bugzilla.redhat.com/show_bug.cgi?id=2237895

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@SanjalKatiyar Please review

**Comment by SanjalKatiyar** [comment]
> @SanjalKatiyar Please review

please check with UX team...

**Comment by SanjalKatiyar** [comment]
> > @SanjalKatiyar Please review
> 
> please check with UX team...

also, I can see need_info on the BZ, plz get those things resolved and clarified...

**Comment by SanjalKatiyar** [comment]
/hold
1. check whether the capacity that we show on the block-pool card is for entire cluster capacity (absolute, including storage needed for replication) or the usable one ?
2. If it is usable then, check https://bugzilla.redhat.com/show_bug.cgi?id=2185042#c17 and confirm if we need to change the name of the card as well or not, "Raw" word at other places is associated with entire storage (not just usable), if so is it okay to use that term here ?

**Comment by TimothyAsirJeyasing** [comment]
Its for the for entire cluster capacity

**Comment by bipuladh** [comment]
The raw capacity of a pool is for that particular pool and not for all pools. Please update the text accordingly.

**Comment by TimothyAsirJeyasing** [comment]
After the discussion with Eran, UX and with the team, The UX has come up with the text change which also gets approval from Doc team. The suggestion is to change the text to "Raw capacity shows the total physical capacity from all the storage pools in the storage system"

File: locales/en/plugin__odf-console.json [hunk 1]
```diff
@@ -359,7 +359,7 @@
   "Mirroring": "Mirroring",
   "Show image states": "Show image states",
   "Last checked": "Last checked",
-  "Raw Capacity shows the total physical capacity from all storage media within the storage subsystem": "Raw Capacity shows the total physical capacity from all storage media within the storage subsystem",
+  "Raw capacity shows the total physical capacity from all the storage pools in the storage system.": "Raw capacity shows the total physical capacity from all the storage pools in the storage system.",
```
[the remark this answers is not in the document]
Comment (TimothyAsirJeyasing) [reply locales/en/plugin__odf-console.json]: done

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**